### PR TITLE
Migrate alert runbooks for cluster-autoscaler-operator

### PR DESCRIPTION
**Pull request by cblecker**
This migrates the current alert descriptions and suggested resolution paths from https://github.com/openshift/cluster-autoscaler-operator/blob/master/docs/user/alerts.md

### CODE REVIEW

**Comment by jan--f** [comment]
The other runbooks in the repo all follow the same structure: Meaning, Impact, Diagnosis, Mitigation.

I think it would be good if the new runbooks could follow the same structure or is there any concern?

File: alerts/cluster-autoscaler-operator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+component: "Cloud Compute"
+subcomponent: "Other Providers"
+approvers:
+  - enxebre
+  - frobware
+  - michaelgugino
```
Comment (elmiko) [inline alerts/cluster-autoscaler-operator/OWNERS:6]: we can remove `frobware` and `michaelgugino`, and add `lobziik` and `fedosin` please

**Review by elmiko — COMMENTED** [review]
looks mostly good, quick comment about the owners

File: alerts/cluster-autoscaler-operator/OWNERS (hunk 1 above)
Comment (cblecker) [reply alerts/cluster-autoscaler-operator/OWNERS:6]: @elmiko I copied this file from https://github.com/openshift/cluster-autoscaler-operator/blob/master/OWNERS.. should we possibly update things there?

**Comment by cblecker** [comment]
@jan--f You're right, I think consistency makes sense here. I'll take a stab at that (although I might need your help @elmiko).

File: alerts/cluster-autoscaler-operator/OWNERS (hunk 1 above)
Comment (elmiko) [reply alerts/cluster-autoscaler-operator/OWNERS:6]: yeah definitely, i can propose a patch there

Comment (elmiko) [reply alerts/cluster-autoscaler-operator/OWNERS:6]: @cblecker just fyi, this is the updated list https://github.com/openshift/cluster-autoscaler-operator/pull/231

Comment (cblecker) [reply alerts/cluster-autoscaler-operator/OWNERS:6]: perfect! I'll update it here too

**Comment by jan--f** [comment]
@cblecker ping

**Pull request closed without merging**